app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -155,14 +156,23 @@ AcuantContext.displayName = 'AcuantContext';
 /**
  * Returns a found AcuantJavascriptWebSdk
  * object, if one is available.
- * This function normalizes differences between
- * the 11.5.0 and 11.7.0 SDKs. The former attached
- * the object to the global window, while the latter
- * sets the object in the global (but non-window)
- * scope.
+ * Depending on the SDK version,
+ * will use either startWorkers (11.8.2) or start (11.9.1)
  */
 const getActualAcuantJavascriptWebSdk = (): AcuantJavascriptWebSdkInterface => {
-  if (window.AcuantJavascriptWebSdk) {
+  if (
+    window.AcuantJavascriptWebSdk &&
+    typeof window.AcuantJavascriptWebSdk.startWorkers === 'function' &&
+    typeof window.AcuantJavascriptWebSdk.start !== 'function'
+  ) {
+    return {
+      ...window.AcuantJavascriptWebSdk,
+      start(...args) {
+        window.AcuantJavascriptWebSdk.startWorkers?.(...args);
+      },
+    };
+  }
+  if (window.AcuantJavascriptWebSdk && typeof window.AcuantJavascriptWebSdk.start === 'function') {
```

Comment:
Minor thing, should we move this up before line 163? since 11.9.1 will be the most cases now.

Reply:
I think it's six of one, half a dozen of the other.  If you want, I can change it.